.browserslistrc

```diff
@@ -0,0 +1 @@
+> 0.5%, last 2 versions, Firefox ESR, not dead
```

Comment:
This doesn't quite match our policy: https://github.com/vector-im/element-web#supported-environments is a lot more strict than this

Reply:
Makes sense, I've updated the rules, here's a list of the matching browsers. 
It looks like there's an issue with their tool and OperaMini isn't excluded properly

<img width="681" alt="Screenshot 2022-11-10 at 16 23 05" src="https://user-images.githubusercontent.com/769871/201150739-0deb0e8f-7106-4f21-b959-cdccdee17fef.png">

Reply:
Is there any way to do it without using exclusions? Right now this means that if a new browser comes around (like the one you DM'd me about) and browserlist supports it then it will wrongly claim support for the browser when our support policy won't be considering it

Reply:
No. https://github.com/browserslist/browserslist/issues/91

Reply:
<img width="875" alt="image" src="https://user-images.githubusercontent.com/2403652/201164193-ffd1bb70-eacc-4ae8-be97-1b5723e9d5e9.png">

This looks to match our support policy better

Reply:
Yes, which seems like the right thing to do, given we have about 6% of users on Chrome 105

Reply:
It'd require invoking product to update the policy we triage against

Reply:
Hmm, in which case, given we use Modernizr for feature detection shouldn't we instead be targeting an ES version?